xml_inscriptions/transcribed/MO.Col.UM.MAA.L.70.161.xml

```diff
@@ -0,0 +1,120 @@
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc hands="1">
+                     <handNote ana="#stamped"> </handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote>
+                        <p>two figures leaning over a basin</p>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history><!--summary, origin, provenance go here-->
+                  <origin>
+                     <date evidence="#unspecified" notBefore="0101" notAfter="0300">second to third
+                        century CE</date>
+                     <placeName ref="europe.italy.rome">unknown</placeName>
+                  </origin>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CIL_XV"/>
+                  <biblScope unit="item">6609a</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US&amp;g_AAEM=70.161"
+                     >Museum of Art and Archeology</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include  href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2021-07-23" who="Sophia Papandonatou">added some metadata, trancription, translation, and museum link</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab>
+               <lb n="1"/><persName><name type="praenomen" key="Lucius"><expan><abbr>L</abbr><ex>uci</ex></expan></name> <name type="gentilicium"><expan><abbr>Pas</abbr></expan></name> <name key="Isidorus" type="cognomen"><expan><abbr>Isid</abbr><ex cert="low">ori</ex></expan></name></persName>.
+            </ab>
+         </div>
+         
+         <div type="translation">
+            <p>Of Lucius Pas( ) Isidorus.</p>
+         </div>
+         
+         <div type="commentary">
+            <p>L(uci) Pas(idi) Isid(ori) (ECortonese 00149).</p>
```

Comment:
Does this reference appear in the bibl? Since I'm pretty sure it's not CIL XV, right?

Reply:
[This](https://www.abebooks.com/Epigrafia-cortonese-Testi-greci-romani-Estratto/30888324634/bd) is the book being referenced. I wasn't sure how to write up the bibliographic entry; it seems like an article within a volume of an annual publication:
```
  <bibl type="j" xml:id="AAEC">
      <abbr type="primary">AAEC</abbr>
      <title>Annuario / Accademia Etrusca di Cortona.</title>
      <pubPlace>Cortona</pubPlace>
   </bibl>

   <bibl type="a-j" xml:id="AAEC_1965-1967_Donati">
      <author>
         <persName type="sort">Donati, Angela</persName>
      </author>. <title level="j" ref="#AAEC">Annuario / Accademia Etrusca di Cortona</title>,
                <biblScope unit="vol" from="13">vol. XIII</biblScope>, <date when="**1965 1967**">1965-1967</date>. </bibl>
```

Would I need a volume entry as well?

Reply:
OK, I see what's going on here now. Yes, we should probably add a volume (though @emylonas should weigh in since I know she's changed how some of those things link together; I can't remember if it's the case that we do need volumes of individual articles anymore, but I think we do). The entry as you've roughed it out here seems generally fine, though just note that you don't need "vol." in front of XIII, and for the dating you could use notBefore and notAfter instead of when.